src/main/java/uk/gov/pay/adminusers/model/Invite.java

```diff
@@ -14,23 +15,23 @@
 @JsonInclude(Include.NON_NULL)
 public class Invite {
 
+    private String code;
     private final String email;
     private String telephoneNumber;
     private Boolean disabled = Boolean.FALSE;
     private Integer loginCounter = 0;
```

Comment:
For someone that see loginCounter in an Invite will get very confused. Maybe we could think about another name (no for this PR, just to discuss if you agree)